keel-core/src/main/java/com/netflix/rocket/semver/DebianVersionComparator.java

```diff
@@ -0,0 +1,123 @@
+package com.netflix.rocket.semver;
```

Comment:
thoughts on calling this package `com.netflix.rocket.semver.shaded` or something so if this becomes a class in its own library we can import without silent conflict?

Reply:
Seems legit